package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "universalviewer",
-  "version": "4.0.25",
+  "version": "4.1.0",
```

Comment:
Something else to discuss on the Community Call: exactly how should we apply semantic versioning, and at what point in time? Does it make sense to increment the version in a PR, or should there be some other process for determining if/when this happens?